bqat-sdk-default.properties

```diff
@@ -0,0 +1,13 @@
+# bqat sdk service used as a quality service for all modalities 
+# bqat server ipaddress running using seperate docker
+bqat.server.ipaddress=91.203.134.4
```

Comment:
Should we hardcode this IP here or change it to a loopback IP? @JanardhanBS-SyncByte

Reply:
let it be localhost, devops team will change the ipaddress and deploy

Reply:
Done